index.html

```diff
@@ -131,8 +122,8 @@ <h1>Education</h1>
                     <h2>freeCodeCamp</h2>
                     <p class="subDetails">January 2019 - present</p>
                     <p><ul>
-                          <li>Responsive Web Design Certificate</li>
-                          <li>JavaScript Algorithms and Data Structures Certificate</li>
+                          <li>Responsive Web Design Certificate. Built five webpages with HTML5/CSS3.</li>

```

Comment:
Links? Just call them out as examples. For example, instead of saying, "I made five pages," say, "Demonstrates real-world page design from scratch: (link link link..)"